ui/crypto-commands.go

```diff
@@ -579,7 +579,7 @@ func cmdCrossSigningGenerate(cmd *Command, container ifc.MatrixContainer, mach *
 		if !uia.HasSingleStageFlow(mautrix.AuthTypePassword) {
 			for _, flow := range uia.Flows {
 				if len(flow.Stages) != 1 {
-					return nil
+					continue
```

Comment:
I have changed this to my own discretion. It might not be correct.
The original code is definitely incorrect.